lib/gumboot/shared_examples/foreign_keys.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.shared_examples 'Foreign Keys' do
+  conn = ActiveRecord::Base.connection
+  context 'Permissions Foreign Keys', if: conn.supports_foreign_keys? do
+    let(:permission_fk) do
+      conn.foreign_keys(:permissions).find do |fk|
+        fk.options[:column] == 'role_id'
+      end
```

Comment:
The pattern on L5 - L7 is duplicated 5 times in this file, you should be able to abstract this to a single method that takes in `table_name` and `column`.